src/cript/api/utils/_get_host_and_token.py

```diff
@@ -0,0 +1,58 @@
+import json
```

Comment:
I saw that you used `utils/*.py` in the other branch about the identifiers too.

Can we please consolidate this with `util.py`?
Having `util` and `utils` as separate things is confusing.


I resolved this in the validate node branch already like this: b697be6e4315ba31d8

Reply:
do you want to have a utils directory with separate utils files or are you thinking a single utils.py file for everything?

Reply:
do you want to have a utils directory with separate utils files or are you thinking a single utils.py file for everything?

Reply:
What I did is, I renamed the `utils` directory into `util`  and the `util.py` file into `util/__init__.py`.
Making the existing `util.py` the core of the new `util` module. (Hence imports of the old stuff doesn't change.)

And from there we can add more files into the module.
Seems like a clean approach to me.

Reply:
I see that you have utils in `src/cript/node/utils/` are you thinking of putting the api utils in that file?

Reply:
No, I was thinking that this module is strictly only for nodes.

But we can mimic that structure for the API as well.

Reply:
ooooh okay gotcha! okay I'll do that for api util folder too!